### DISCUSSION TITLE
Conflict library walletconnect v2 with solana in Dapp

### DISCUSSION THREAD

**Discussion by johnnydepp9** [WalletConnect/walletconnect-monorepo#3153] [Developer Support]
I use react 17, node 16 to upgrade v2.
In my react app using @solana/wallet-adapter-wallets. Maybe conflict with these library of v2

Failed to compile.
./node_modules/@ledgerhq/hw-transport-webhid/lib-es/TransportWebHID.js
Module not found: Can't resolve '@ledgerhq/devices/hid-framing' in '/home/sotatek/Desktop/my-app/node_modules/@ledgerhq/hw-transport-webhid/lib-es'

**Comment by johnnydepp9** [comment]
I using web3React, and my website support Sollet and Phantom Wallet in solana chain.

**Comment by ordec2** [comment]
nono
